custom/tests.yaml

```
@@ -3284,17 +3311,28 @@ api:
         return false;
       }
 
+      // The following function is defined to help obtain individual stats types
+      // and entries
+      function getStatByTypeAndEntry(type, entry) {
+        for (var i = 0; i < stats.length; i++) {
+          if (stats[i].type === type
+            && Object.prototype.hasOwnProperty.call(stats[i], entry)
+            && stats[i][entry] !== null) {
```

I think we can simplify this by doing the following:

```suggestion
          if (stats[i].type === type && !!stats[i][entry]) {
```

---

Thank you! I may write un-Idomatic JavaScript code because I don't use it every day.